guides/araujo/FTP Writeup.md

```diff
@@ -0,0 +1,92 @@
+</ul>
+<h3 id="table">Table</h3>
+<ol>
+<li><a href="#id-link-to-section">Prerequisites</a></li>
+<li><a href="#id0">Summary</a></li>
+<li><a href="#id1">Setting up and Installing Windows Server 2003</a></li>
+<li><a href="#id2">Creating and Configuring an FTP Server</a></li>
+</ol>
+<h2 id="prerequisites-a-idid-link-to-sectiona">Prerequisites <a id="id-link-to-section"></a></h2>
+<ol>
+<li>Have your <a href="https://silexone.github.io/guides/nestor/ISPsetup.html">virtual environment</a> configured</li>
+<li>Have the <a href="https://silexone.github.io/guides/william/DNSNotes.html">DNS</a> machine running.</li>
+<li>Have <a href="link-to-guide">pfSense</a> running.</li>
+</ol>
+<h2 id="summary-a-idid0a">Summary <a id="id0"></a></h2>
+<p>This guide features the initial setup of the virtual machine, installation of Windows Server 2003, and the creation of FTP servers along with configuring and securing the servers.</p>
+<h2 id="setting-up-and-installing-windows-server-2003-a-idid1a">Setting up and Installing Windows Server 2003 <a id="id1"></a></h2>
+<p>Before you can setup your FTP server, you must download this ISO file to install the virtual machine.  Once downloaded, open VirtualBox and click on “New” at the top left.  Go through the initial setup for creating the virtual machine similarly to the way you did with the first few boxes.  Just make sure to allow yourself enough memory to be able to work in the environment properly.</p>
+<p>Before starting the virtual machine, make sure it is selected and click on “Settings”.  Once you’re in settings, navigate to the “Network” tab and under “Adapter 1”, select “Host-Only Adapter” and choose the adapter representing the LAN.</p>
+<p>Once the options are completed, make sure your new server is selected and click start at the top.  Once started, make sure to select the proper ISO file from where you downloaded it from.</p>
+<p>Choose the default options for these first few pages and let the installation process begin.</p>
+<p><img src="/pics/cap%208.PNG" alt=""></p>
+<p>Once promoted, fill in the boxes with appropriate info and continue through the settings and make sure to set an admin password.</p>
+<p><img src="/pics/cap%209.PNG" alt=""></p>
+<p>Instead of choosing “Typical settings”, click on “Custom settings” and click next.</p>
+<p><img src="/pics/cap%2010.PNG" alt=""></p>
+<p>Select the “Internet Protocol (TCP/IP)” option and click on properties.</p>
+<p>Once you’re at this screen, make sure to select the manual options for both IP addresses and DNS server addresses and input these addresses as follows:<br>
+<br></p>
+
+<table>
+<thead>
+<tr>
+<th>Name</th>
+<th>IP Address</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>IP Address</td>
+<td>172.20.241.9</td>
+</tr>
+<tr>
+<td>Subnet Mask</td>
+<td>255.255.255.0</td>
+</tr>
+<tr>
+<td>Default Gateway</td>
+<td>172.20.241.254</td>
+</tr>
+<tr>
+<td>Preferred DNS Server</td>
+<td>8.8.8.8</td>
+</tr>
+</tbody>
```

Comment:
Markdown would make this look a lot better. Though this is awesome when displayed on your page, easy to read in a table.